yarn.lock

```diff
@@ -9758,10 +9758,10 @@ react-native-xaml@^0.0.63:
     "@types/react-native" "*"
     typescript "^4.4.3"
 
-react-native@0.69.0-rc.6:
-  version "0.69.0-rc.6"
-  resolved "https://registry.yarnpkg.com/react-native/-/react-native-0.69.0-rc.6.tgz#7e344d0c79ae7bd8fc7e00ec0ff857a51a6e6abb"
-  integrity sha512-TrzKCtaQuqR3Pm5pc9i2zHOcBnAuJewwvp0B/BL9hvd/kYQLdb7QYzoJJ3KPv/9HRj9Ma95mZE01v8dVNfRbXg==
+react-native@0.69.0, react-native@^0.69.0:
```

Comment:
Why is this called out twice?

Reply:
I believe that's because some items in the repo are on ^0.69.0 i.e. the rnw package itself. The precedent we've set for other packages like playground is for them to stay on simply 0.69.0. Thus, for the one yarn.lock file in the repo both dependencies are mentioned and 0.69.0 is installed.